frontend/packages/schema-editor/src/components/SchemaInspector/ItemFieldsTab/ItemFieldsTable/ItemFieldsTableRow.tsx

```diff
@@ -82,16 +82,14 @@ export const ItemFieldsTableRow = ({
           handleSave={handleChangeNodeName}
           onKeyDown={onKeyDown}
           pointer={fullPath}
+          size='small'
           aria-label={t('schema_editor.field_name')}
         />
       </td>
       <td className={cn(classes.tableColumnType, classes.tableCell)}>
-        <Select
-          hideLabel
-          inputId={`${fieldNode.domId}-typeselect`}
-          label={t('schema_editor.type')}
-          onChange={(fieldType) => onChangeType(fullPath, fieldType as FieldType)}
-          options={getTypeOptions(t)}
+        <TypeSelect
+          id={`${fieldNode.domId}-typeselect`}
+          onChange={(fieldType) => onChangeType(fullPath, fieldType)}
```

Comment:
Hvis jeg skal pirke på småtteri, så hadde jeg kanskje endret navnet på denne? 😊  
```suggestion
          onChange={(fieldType) => onTypeChange(fullPath, fieldType)}
```

Reply:
Enig, godt forslag!